src/repository/users.py

```diff
@@ -1,9 +1,13 @@
+
```

Comment:
Is that automatically added? I have syntax error in PyCharm with empty line here